src/server/pkg/serviceenv/config.go

```diff
@@ -91,6 +91,9 @@ type WorkerSpecificConfiguration struct {
 }
 
 // FeatureFlags contains the configuration for feature flags.
+// XXX: if you're adding a new feature flag then you need to make sure it gets
```

Comment:
You might want to make some mention of the worker environment as well. Feature flags may apply to the worker and/or sidecar environments.

Reply:
I'm not sure I totally understand what you're saying here, are you thinking it'd be possible to have feature flags that only apply to the workers? I was thinking that these feature flags were basically global and should show up in `pachd` (including sidecars) as well as the user container in worker pods.

Reply:
I was referring to the environment that we setup for the user container, which has the variable name `workerEnv`. There is probably a better name for that variable. Right now, the feature flags are not getting propagated to the user container, and the comment does not mention the user container environment.

Reply:
Ahh, makes sense. And it looks like the user container already has the `FeatureFlags` struct in its env, it just isn't being propagated to it, I'll update.

Reply:
Done.